resources/scripts/ttx_diff.py

```diff
@@ -115,6 +116,22 @@ def ttx(font_file: Path, can_skip: bool):
     run(cmd, font_file.parent, check=True)
     return ttx_file
 
+# run normalizer on both fonts at once; because this is an external process
+# we can use the threading module
+def run_layout_normalizer(cargo_manifest_path: Path, build_dir: Path, fontc_ttf: Path, fontmake_ttf: Path):
+
+    can_skip_fontc = can_skip(build_dir, "fontc")
+    can_skip_fontmake = can_skip(build_dir, "fontmake")
+    t1 = threading.Thread(target=simple_gpos_output, args = [cargo_manifest_path, fontc_ttf, build_dir / "fontc.markkern.txt", can_skip_fontc])
+    t2 = threading.Thread(target=simple_gpos_output, args = [cargo_manifest_path, fontmake_ttf, build_dir / "fontmake.markkern.txt", can_skip_fontmake])
```

Comment:
This looks sus, why would we need to start a thread to start a process? Just start the processes?

Reply:
https://docs.python.org/3/library/subprocess.html#subprocess.Popen should be able to do non-blocking launches